### DISCUSSION TITLE
never receives email for confirming account

### DISCUSSION THREAD

**Discussion by lepinekong** [community/community#21863] [New to GitHub]
<p>I forgot to resend email for account email verification, I tried several times but still doesn’t receive any. My email works, I tested by sending from another account not on same provider.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/lepinekong">@lepinekong</a>,</p>
<p>If you haven’t already, I would recommend that you check your spam folder or any filter settings that you might have set up. If the emails aren’t caught there, I would recommend <a href="https://github.com/contact" rel="nofollow noopener">contacting support directly</a> to get help verifying your account.</p>
<p>Thanks!</p>

**Comment by lepinekong** [comment]
<p>I had 2 emails I confused one with the other sorry.</p>
